libafl_bolts/src/shmem.rs

```diff
@@ -648,18 +648,18 @@ pub mod unix_shmem {
                         0o600,
                     );
                     if shm_fd == -1 {
-                        perror(b"shm_open\0".as_ptr() as *const _);
+                        let os_error = io::Error::last_os_error();
                         return Err(Error::unknown(format!(
```

Comment:
we can make another err enum Error::UnixOSError

Reply:
for this one maybe.
but for the others that fails at shmat/mmap is not really IO